openbb_terminal/stocks/dark_pool_shorts/finra_view.py

```diff
@@ -207,7 +186,7 @@ def darkpool_otc(
     num: int,
     promising: int,
     tier: str,
```

Comment:
Is there a way to have a default kwarg for the `tier`? It's a string and giving a default kwarg would make it easier to understand what it looks like